target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-ix4-200d.dts

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "kirkwood.dtsi"
+#include "kirkwood-6281.dtsi"
+
+/ {
+	model = "Iomega ix4-200d";
+	compatible = "iom,ix4-200d", "marvell,rd88f6281-a", "marvell,rd88f6281", "marvell,kirkwood-88f6281", "marvell,kirkwood";
+
+memory {
+	device_type = "memory";
+	reg = <0x00000000 0x20000000>;
+};
+
+chosen {
+	bootargs = "console=ttyS0,115200n8 earlyprintk";
+	stdout-path = &uart0;
+};
+
+mbus {
+	pcie-controller {
+		status = "okay";
+
+			pcie@1,0 {
+			status = "okay";
+			};
+		};
+};
+
+ocp@f1000000 {
+	pinctrl: pin-controller@10000 {
+		spi_pin1: spi-pin1 {
```

Comment:
Shrink pin properties into one is easier to read. Please look: https://github.com/openwrt/openwrt/blob/08cc6bc6649f0f184868bd950e38c6107abba0ae/target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-c200-v1.dts#L251